<?xml version="1.0" encoding="utf-8"?>

<launch>

  <arg name="target_height_threshold"		default = "0.05" />
  <arg name="target_width_threshold"		default = "0.05"/>

  <node pkg="pointcloud_cluster_detect" type="pointcloud_cluster_detect" name="pointcloud_cluster_detect" output="screen">
    <param name="target_height_threshold" value="$(arg target_height_threshold)" />
    <param name="target_width_threshold" value="$(arg target_width_threshold)" />
  </node>
</launch>
